### ISSUE TITLE
Looks like the image doesn't work

### ISSUE THREAD

**Issue by vkhmialnitski** [port-labs/port-k8s-exporter#32]
podman run ghcr.io/port-labs/port-k8s-exporter:0.2.6
{"msg":"exec container process (missing dynamic library?) `/usr/bin/port-k8s-exporter`: No such file or directory","level":"error","time":"2024-01-02T09:00:41.655032Z"}  

Same error in AKS via install through the Helm chart:
exec /usr/bin/port-k8s-exporter: no such file or directory

**Comment by eshepelyuk** [comment]
I am receiving the same message while trying to install exporter into a local cluster.
```
exec /usr/bin/port-k8s-exporter: no such file or directory
```

**Comment by eshepelyuk** [comment]
I've tried to run multiple versions - and this one looks like the latest that its working

```
docker run ghcr.io/port-labs/port-k8s-exporter:0.1.14 --help
```

**Comment by yairsimantov20** [comment]
Hi, Thank you for letting us know 😄 

This should be fixed on version 0.2.7 of the helm chart that installs version 0.2.7 for the exporter

Make sure to update your k8s-exporter helm chart to the latest version

@eshepelyuk @vkhmialnitski

**Comment by eshepelyuk** [comment]
Hello @yairsimantov20 
As I've already mentioned - 0.2.7 is not working 

```
 docker run ghcr.io/port-labs/port-k8s-exporter:0.2.7 --help
exec /usr/bin/port-k8s-exporter: no such file or directory
```
